### DISCUSSION TITLE
Quantization

### DISCUSSION THREAD

**Discussion by srevandro** [g588928812/bitsandbytes_jetsonX#3] [General]
Hello, 

How are you? Please, I need some clarification. 

Can the 8-bit optimizer (LLM.int8) be used in Jetson Xavier? On your tests you have reported lots of failures. 

I will buy a Jetson AGX Xavier with 16GB RAM and 32GB SSD. Would it work?

Besides, do you think QLoRA should RAM in the Jetson AGX Xavier? Or is it impossible in any embedded system or mobile device?

Thanks.

**Comment by geronimi73** [comment]
Yes it did work back then even though some tests failed. 
BUT, it was a pain the a. If you can, get a proper computer and buy a RTX 3090 (24 GB VRAM), cheap and you will be able to do almost anything.
